<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="install_kernel_block">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>install_kernel_block</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uninstall_kernel_block">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>uninstall_kernel_block</string>
    </property>
   </widget>
   <widget class="QPushButton" name="create_rule">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>create_rule</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_all_rules">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>show_all_rules</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connect">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>connect init</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>911</width>
      <height>371</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_rule">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>140</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>delete_rule</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nat">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>nat</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>908</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
